pootle/static/css/style.css

```diff
@@ -1160,6 +1166,15 @@ table.stats td.last-updated
     text-align: right;
 }
 
+html[dir="rtl"] table.stats .number,
+html[dir="rtl"] th.stats-number,
+html[dir="rtl"] td.stats-number,
+html[dir="rtl"] td.stats-percentage,
+html[dir="rtl"] table.stats td.last-updated
```

Comment:
I think it's better that "table.stats td.last-updated" is aligned to right, beacuse that column contains RTL text ..